runtime/lua/vim/_watch.lua

```diff
@@ -289,6 +289,9 @@ function M.fswatch(path, opts, callback)
       end
 
       if data and #vim.trim(data) > 0 then
+        if vim.startswith(data, 'Event queue overflow') then
```

Comment:
This probably also needs an OS check.

Reply:
You are right, looks like Windows can also emit this message. Added the chec and updated rest of docs as well, thanks.